samples/js-coffee-shop/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "coffee-shop",
+  "version": "1.0.0",
+  "description": "Samples for a coffeeshop",
+  "main": "lib/index.js",
+  "scripts": {
+    "start": "node lib/index.js",
+    "compile": "tsc",
+    "build": "npm run build:clean && npm run compile",
+    "build:clean": "rm -rf ./lib",
+    "build:watch": "tsc --watch",
+    "build-and-run": "npm run build && node lib/index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@genkit-ai/ai": "~0.5.0-rc.11",
+    "@genkit-ai/core": "~0.5.0-rc.11",
+    "@genkit-ai/dotprompt": "~0.5.0-rc.11",
+    "@genkit-ai/flow": "~0.5.0-rc.11",
+    "genkitx-chromadb": "~0.5.0-rc.11",
+    "@genkit-ai/dev-local-vectorstore": "~0.5.0-rc.11",
+    "@genkit-ai/firebase": "~0.5.0-rc.11",
+    "@genkit-ai/googleai": "~0.5.0-rc.11",
+    "genkitx-ollama": "~0.5.0-rc.11",
```

Comment:
This makes sense to me, thinking through the process, if someone is just testing they can `genkit start` here and build and run using the published versions.  If they're doing local development the best process would be to bump their local `package.json` file to the next patch/RC version and the validate that it's getting picked up in these samples, but they shouldn't have to adjust the imports since they're getting ~'d in, is that right?